atoman/gui/dialogs/infoDialogs.py

```diff
@@ -679,8 +682,10 @@ def __init__(self, pipelinePage, atomIndex, scalarsDict, vectorsDict, filterList
         self.filterList = filterList
         self.neighbourInfoWindow = None
         
-        lattice = self.pipelinePage.inputState
+        self.PBC = [1,1,1]
```

Comment:
Is there a reason to force periodic boundaries to be on? Could we use `lattice.PBC` instead? This should have the correct PBCs for the lattice (the user can set them).